telemetry/main.schema.json

```diff
@@ -27,11 +27,15 @@
         },
         "platformVersion": {
           "type": "string",
-          "pattern": "^[0-9]{2}\\."
+          "pattern": "^[0-9]{3}\\."
```

Comment:
Won't this now fail for 2-digit versions?  I am in favour of `"^[0-9]+\\."`, but if you want to limit to 2 or 3 digits only, we should use `"^[0-9]{2,3}\\."`